deployment/united-manufacturing-hub/Chart.yaml

```diff
@@ -18,13 +18,13 @@ type: application
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 
-version: 0.9.9
+version: 0.9.10-prerelease1
```

Comment:
unintendet right?

Reply:
This is going to be overwritten once this pr is merged into main after 0.9.10 is merged into main. Worst case a merge conflict raises and is easily solvable